kaggle_environments/core.py

```diff
@@ -301,7 +301,7 @@ def render(self, **kwargs):
             html = f'<iframe srcdoc="{player_html}" width="{width}" height="{height}" frameborder="0"></iframe> '
             display(HTML(html))
         elif mode == "json":
-            return json.dumps(self.toJSON(), sort_keys=True)
+            return json.dumps(self.toJSON(), sort_keys=True, indent=2 if self.debug else None)
```

Comment:
This will now pretty print replays in debug mode -- it made testing this feature a lot easier.